xclim/ensembles/_robustness.py

```diff
@@ -399,10 +413,10 @@ def robustness_coefficient(
 
     Parameters
     ----------
-    fut : Union[xr.DataArray, xr.Dataset]
-        Future ensemble values along 'realization' and 'time' (nr, nt). Can be a dataset,
-        in which case the coefficient is computed on each variable.
-    ref : Union[xr.DataArray, xr.Dataset]
+    fut : xr.DataArray or  xr.Dataset
+        Future ensemble values along 'realization' and 'time' (nr, nt).
+        Can be a dataset, in which case the coefficient is computed on each variable.
+    ref : xr.DataArray or xr.Dataset
         Reference period values along 'time' (nt). Same type as `fut`.
```

Comment:
```suggestion
    fut : xr.DataArray |  xr.Dataset
        Future ensemble values along 'realization' and 'time' (nr, nt).
        Can be a dataset, in which case the coefficient is computed on each variable.
    ref : xr.DataArray | xr.Dataset
        Reference period values along 'time' (nt). Same type as `fut`.
```

Reply:
The spec suggests using plaintext descriptions here: https://numpydoc.readthedocs.io/en/latest/format.html#parameters.